spec/feature/add_listing_spec.rb

```diff
@@ -1,5 +1,12 @@
 feature 'add listing' do
```

Comment:
can I suggest we split the feature test for 'adding a listing' to include features for 'signup' and 'signin':

e.g.:
featuren'add listing' do
  feature 'signup'
  end

  feature signin
  end
end